src/EditorFeatures/CSharpTest/CodeActions/ReplaceMethodWithProperty/ReplaceMethodWithPropertyTests.cs

```diff
@@ -2360,6 +2360,57 @@ int Goo1
 }");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsReplaceMethodWithProperty)]
+        [WorkItem(42698, "https://github.com/dotnet/roslyn/issues/42698")]
+        public async Task TestMethodWithTrivia_3()
+        {
+            await TestInRegularAndScript1Async(
+@"class C
+{
+    [||]int FooBar() //Vital Comment
```

Comment:
cannot use `Foo`.  either call the method `M`, or `GooBar` or `Goo`

Reply:
I have updated the name